ShareClient/ShareClient.swift

```diff
@@ -79,17 +78,15 @@ public class ShareClient {
     private let shareServer:String
     private var token: String?
 
-    public init(username: String, password: String, shareServer:ShareServer=ShareServer.US) {
+    public init(username: String, password: String, shareServer:String=KnownShareServers.US.rawValue) {
```

Comment:
nit: since the convenience initializer below has a default value for `shareServer`, I don't think there should be a default value here.